docs/execution-providers/TensorRT-ExecutionProvider.md

```diff
@@ -27,21 +27,24 @@ See [Build instructions](../build/eps.md#tensorrt).
 
 ## Requirements
 
-| ONNX Runtime | TensorRT | CUDA       |
-| :----------- | :------- | :--------- |
-| 1.18-main    | 10.0     | 11.8, 12.2 |
-| 1.17         | 8.6      | 11.8, 12.2 |
-| 1.16         | 8.6      | 11.8       |
-| 1.15         | 8.6      | 11.8       |
-| 1.14         | 8.5      | 11.6       |
-| 1.12-1.13    | 8.4      | 11.4       |
-| 1.11         | 8.2      | 11.4       |
-| 1.10         | 8.0      | 11.4       |
-| 1.9          | 8.0      | 11.4       |
-| 1.7-1.8      | 7.2      | 11.0.3     |
-| 1.5-1.6      | 7.1      | 10.2       |
-| 1.2-1.4      | 7.0      | 10.1       |
-| 1.0-1.1      | 6.0      | 10.0       |
+Note: starting ORT 1.19, **CUDA 12** becomes default version when distributing ONNX Runtime GPU packages.
+
+| ONNX Runtime | TensorRT | CUDA           |
+| :----------- | :------- | :------------- |
+| 1.19-main    | 10.2     | **12.2**, 11.8 |
```

Comment:
TensorRT was updated to 10.3 in main.

Reply:
i don't know how scalable it is to keep updating the minor version in these docs.
technically TensorRT and CUDA are minor version compatible so would it be easier to list 10.x and 12.x ?